Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1558,6 +1594,9 @@ bool CStaticFunctionDefinitions::DetachElements(CElement* pElement, CElement* pA
             BitStream.pBitStream->Write(vecPosition.fX);
             BitStream.pBitStream->Write(vecPosition.fY);
             BitStream.pBitStream->Write(vecPosition.fZ);
+            BitStream.pBitStream->Write(vecRotation.fX);
+            BitStream.pBitStream->Write(vecRotation.fY);
+            BitStream.pBitStream->Write(vecRotation.fZ);
```

Comment:
Is element rotation broken after  attachElements?
If there is a bug, create a new PR with this fix.
+ You need to update bitstream version and add a version check here. Otherwise, it would break clients with old client version

Reply:
Yes, there is a bug with detachElements that the rotation is not retained, only the position. But I have to query the rotation in this PR anyway, because it is necessary for the arguments of the event. So if I make a new PR, I have to wait for this to be merged first, should I really make a new PR for these 3 lines? Version management is fine, I will implement it soon.